source/cypress/fixtures/templaterepo/cdk/packages/compliant-s3-bucket/README.md

```diff
@@ -0,0 +1,11 @@
+# `compliant-s3-bucket`
+
+> TODO: description
+
+## Usage
+
+```
+const compliantS3Bucket = require('compliant-s3-bucket');
+
+// TODO: DEMONSTRATE API
```

Comment:
TODO?

Reply:
This is a template snippet for the users. The TODO is for users to add the required code to demonstrate the API.